code/modules/admin/admin.dm

```diff
@@ -483,6 +483,14 @@ var/global/floorIsLava = 0
 			<A href='?src=\ref[src];secretscoder=infinite_sec'>Remove cap on security officers</A><BR>
 			<BR>
 			"}
+	if(check_rights(R_FUN,0) && check_rights(R_DEBUG,0))

```

Comment:
Oh wow. I can't believe I missed this, actually. Redundant check. Needs to be removed.